src/components/travel-search-filter/types.ts

```diff
@@ -0,0 +1,1643 @@
+import { z } from 'zod';
```

Comment:
Should zod objects/definitions be placed in separate files or is it okay to have them with the types?

Reply:
For cases where you have both zod and definitions I try to use zod and `z.infer` to create types so we only have one source of truth.